libs/langchain/langchain/chains/__init__.py

```diff
@@ -17,135 +17,153 @@
+    {"name": "LLMRequestsChain", "module": "langchain.chains.llm_requests"},
+    {
+        "name": "LLMSummarizationCheckerChain",
+        "module": "langchain.chains.llm_summarization_checker.base",
+    },
+    {"name": "load_chain", "module": "langchain.chains.loading"},
+    {"name": "MapReduceChain", "module": "langchain.chains.mapreduce"},
+    {"name": "OpenAIModerationChain", "module": "langchain.chains.moderation"},
+    {"name": "NatBotChain", "module": "langchain.chains.natbot.base"},
+    {
+        "name": "create_citation_fuzzy_match_chain",
+        "module": "langchain.chains.openai_functions",
+    },
+    {"name": "create_extraction_chain", "module": "langchain.chains.openai_functions"},
+    {
+        "name": "create_extraction_chain_pydantic",
+        "module": "langchain.chains.openai_functions",
+    },
+    {
+        "name": "create_qa_with_sources_chain",
+        "module": "langchain.chains.openai_functions",
+    },
+    {
+        "name": "create_qa_with_structure_chain",
+        "module": "langchain.chains.openai_functions",
+    },
+    {"name": "create_tagging_chain", "module": "langchain.chains.openai_functions"},
+    {
+        "name": "create_tagging_chain_pydantic",
+        "module": "langchain.chains.openai_functions",
+    },
+    {"name": "QAGenerationChain", "module": "langchain.chains.qa_generation.base"},
+    {"name": "QAWithSourcesChain", "module": "langchain.chains.qa_with_sources.base"},
+    {
+        "name": "RetrievalQAWithSourcesChain",
+        "module": "langchain.chains.qa_with_sources.retrieval",
+    },
+    {
+        "name": "VectorDBQAWithSourcesChain",
+        "module": "langchain.chains.qa_with_sources.vector_db",
+    },
+    {"name": "create_retrieval_chain", "module": "langchain.chains.retrieval"},
+    {"name": "RetrievalQA", "module": "langchain.chains.retrieval_qa.base"},
+    {"name": "VectorDBQA", "module": "langchain.chains.retrieval_qa.base"},
+    {"name": "LLMRouterChain", "module": "langchain.chains.router"},
+    {"name": "MultiPromptChain", "module": "langchain.chains.router"},
+    {"name": "MultiRetrievalQAChain", "module": "langchain.chains.router"},
+    {"name": "MultiRouteChain", "module": "langchain.chains.router"},
+    {"name": "RouterChain", "module": "langchain.chains.router"},
+    {
+        "name": "SequentialChain",
+        "module": "langchain.chains.sequential",
+    },
+    {
+        "name": "SimpleSequentialChain",
+        "module": "langchain.chains.sequential",
+    },
+    {"name": "create_sql_query_chain", "module": "langchain.chains.sql_database.query"},
+    {
+        "name": "create_structured_output_runnable",
+        "module": "langchain.chains.structured_output",
+    },
+    {"name": "load_summarize_chain", "module": "langchain.chains.summarize"},
+    {"name": "TransformChain", "module": "langchain.chains.transform"},
 ]
+
+_module_lookup = {entry["name"]: entry["module"] for entry in _exports}
```

Comment:
why not just make _exports be a dict

Reply:
Oh this was from `llms` where we also had an `llm_type` key to store, so had two different things I had to iterate over

Will reformat here though